ts/features/wallet/onboarding/paypal/screen/PayPalPspSelectionScreen.tsx

```diff
@@ -117,20 +103,38 @@ const PayPalPspSelectionScreen = (props: Props): React.ReactElement | null => {
     280
   );
   const pspList = getValueOrElse(props.pspList, []);
-  const [selectedPsp, setSelectedPsp] = useState<
-    IOPayPalPsp["id"] | undefined
-  >();
-
+  const [selectedPsp, setSelectedPsp] = useState<IOPayPalPsp | undefined>();
   const dispatch = useDispatch();
   const searchPaypalPsp = () => {
     dispatch(searchPaypalPspAction.request());
```

Comment:
> We recommend using the React-Redux hooks API as the default approach in your React components.
https://react-redux.js.org/api/hooks
Since recently we have upgraded the redux library I prefer to use this less verbose and more component oriented hook